packages/predicates/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "@shopify/predicates",
+  "version": "0.0.0",
+  "license": "MIT",
+  "description": "A set of common predicates for JS/TS apps",
```

Comment:
I'd drop the acronyms and the "apps". What about `A set of common Javascript predicates`?